groups/instance/security_groups.tf

```diff
@@ -35,3 +35,39 @@ resource "aws_security_group" "instance_security_group" {
   }
 
 }
+
+resource "aws_security_group" "alb_security_group" {
+  name        = "${var.environment}-${var.service}-lb"
+  description = "Restricts access for ${var.service}-${var.environment} lb artifactory nodes"
+  vpc_id      = data.aws_vpc.placement.id
+
+  ingress {
+    description      = "lb HTTP ingress from admin and concourse CIDRs"
+    from_port        = 80
+    to_port          = 80
+    protocol         = "tcp"
+    cidr_blocks      = local.artifactory_web_access
+  }
+
+  ingress {
+    description      = "lb HTTPS ingress from admin and concourse CIDRs"
+    from_port        = 443
+    to_port          = 443
+    protocol         = "tcp"
+    cidr_blocks      = local.artifactory_web_access
+  }
```

Comment:
Ideally we'd also include the administration prefix list for ingress access as well by additionally setting the `prefix_list_ids` parameter alongside `cidr_blocks`